Cargo.toml

```diff
@@ -88,13 +88,15 @@ async-trait = { version = "~0.1.30", optional = true }
 torut = { version = "~0.1.6", features = ["v2", "v3"] }
 
 [target.'cfg(target_os="android")'.dependencies]
+cc = { version = "=1.0.41" }
+socket2 = { version = "=0.3.15" }
 zmq = { version = "~0.9", features = ["vendored"] }
 
 [target.'cfg(target_os="ios")'.dependencies]
-zeromq-src = { git = "https://github.com/LNP-BP/zeromq-src-rs", branch = "fix/cmake" }
+zeromq-src = { version = "~0.1", git = "https://github.com/LNP-BP/zeromq-src-rs", branch = "fix/cmake" }
```

Comment:
Why do we need the version part? I thought it is incompatible/not working when you also provide git and branch

Reply:
that's something this pre-publishing check uncovered
without version specification, `cargo publish --dry-run` returns the following error:
```
error: all dependencies must have a version specified when publishing.
dependency `zeromq-src` does not specify a version
Note: The published dependency will use the version from crates.io,
the `git` specification will be removed from the dependency declaration.
```